kogito-bom/pom.xml

```diff
@@ -53,2739 +53,1251 @@
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb-deployment</artifactId>
-        <version>${project.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb-deployment</artifactId>
+        <artifactId>kogito-quarkus-workflow-common</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-workflow-common</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-workflow-common-deployment</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-process-management</artifactId>
-        <version>${project.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-process-management</artifactId>
+        <artifactId>kogito-quarkus-workflow-common-deployment</artifactId>
         <version>${project.version}</version>
-        <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-process-management</artifactId>
+        <artifactId>kogito-quarkus-serverless-workflow</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-process-management</artifactId>
+        <artifactId>kogito-quarkus-serverless-workflow</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-process-management-deployment</artifactId>
+        <artifactId>kogito-quarkus-serverless-workflow-deployment</artifactId>
         <version>${project.version}</version>
       </dependency>
```

Comment:
Missing sources for this artifact

Reply:
good point, this would need to be fixed in upstream too, but I guess not a blocker.